airflow-core/src/airflow/ui/src/layouts/Details/Gantt/Gantt.tsx

```diff
@@ -0,0 +1,291 @@
+        duration: 100,
+      },
+      indexAxis: "y" as const,
+      maintainAspectRatio: false,
+      plugins: {
+        annotation: {
+          annotations:
+            selectedId === undefined
+              ? []
+              : [
+                  {
+                    backgroundColor: selectedItemColor,
+                    borderWidth: 0,
+                    drawTime: "beforeDatasetsDraw" as const,
+                    type: "box" as const,
+                    xMax: "max" as const,
+                    xMin: "min" as const,
+                    yMax: data.findIndex((dataItem) => dataItem.y === selectedId) + 0.5,
+                    yMin: data.findIndex((dataItem) => dataItem.y === selectedId) - 0.5,
+                  },
+                ],
+        },
+        legend: {
+          display: false,
+        },
+        tooltip: {
+          callbacks: {
+            afterBody(tooltipItems: Array<TooltipItem<"bar">>) {
+              const taskInstance = data.find((dataItem) => dataItem.y === tooltipItems[0]?.label);
+              const startDate = formatDate(taskInstance?.x[0], selectedTimezone);
+              const endDate = formatDate(taskInstance?.x[1], selectedTimezone);
+
+              return [
+                `${translate("startDate")}: ${startDate}`,
+                `${translate("endDate")}: ${endDate}`,
+                `${translate("duration")}: ${getDuration(taskInstance?.x[0], taskInstance?.x[1])}`,
+              ];
+            },
+            label(tooltipItem: TooltipItem<"bar">) {
+              const { label } = tooltipItem;
+              const taskInstance = data.find((dataItem) => dataItem.y === label);
+
+              return `${translate("state")}: ${translate(`states.${taskInstance?.state}`)}`;
+            },
+          },
+        },
+      },
+      resizeDelay: 100,
+      responsive: true,
+      scales: {
+        x: {
+          grid: {
+            color: gridColor,
+            display: true,
+          },
+          max: formatDate(selectedRun?.end_date, selectedTimezone),
+          min: formatDate(selectedRun?.start_date, selectedTimezone),
+          position: "top" as const,
+          stacked: true,
+          ticks: {
+            align: "start" as const,
+            callback: (value: number | string) => formatDate(value, selectedTimezone, "HH:mm:ss"),
+            maxRotation: 8,
+            maxTicksLimit: 8,
+            minRotation: 8,
+          },
+          type: "time" as const,
+        },
+        y: {
+          grid: {
+            color: gridColor,
+            display: true,
+          },
+          stacked: true,
+          ticks: {
+            display: false,
+          },
+        },
+      },
+    }),
+    [data, selectedId, selectedItemColor, gridColor, selectedRun, selectedTimezone, translate],
+  );
+
+  if (runId === undefined) {
+    return undefined;
+  }
+
+  return (
+    <Box height={`${fixedHeight}px`} minW="500px" ml={-2} mt={36} w="100%">
```

Comment:
```suggestion
    <Box height={`${fixedHeight}px`} minW="200px" ml={-2} mt={36} w="100%">
```

Looking great let's just change the minW and looks good to me

Reply:
Thanks!